web_company_logo_unclickable/static/src/xml/base.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<templates id="template" xml:space="preserve">
+    <t t-extend="WebClient">
+        <t t-jquery="a[class=oe_logo]" t-operation="replace">
```

Comment:
I tried to, but I need to keep "Edit Company Data" the way it is, just remove the clickable link from the logo, so any ideas to make it better?